io-engine/src/core/env.rs

```diff
@@ -1088,6 +1125,11 @@ impl MayastorEnvironment {
         // ensure we are within the context of a spdk thread from here
         Mthread::primary().set_current();
 
+        // To enable SPDK tracing set self.num_entries (eg. to 32768).
+        if self.num_entries > 0 {
+            self.init_spdk_tracing();
```

Comment:
Pardon my ignorance, this num_entries existing already?
What is this tracing actually doing here?

Reply:
`num_entries` already exists in `MayastorEnvironment` which loosely resembles `spdk_app_opts` and indicates the number of trace entries per core.
If this `num_entries` > 0 we initialize the SPDK tracing framework.